x-pack/test/cloud_security_posture_functional/agentless/create_agent.ts

```diff
@@ -68,7 +69,11 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
       expect(await cisIntegration.getFirstCspmIntegrationPageAgentlessIntegration()).to.be(
         integrationPolicyName
       );
-      expect(await cisIntegration.getFirstCspmIntegrationPageAgentlessStatus()).to.be('Pending');
+
+      // wait for eventually healthy status
+      await retry.tryForTime(5000, async () => {
+        expect(await cisIntegration.getFirstCspmIntegrationPageAgentlessStatus()).to.be('Healthy');
```

Comment:
Did you try running this against an actual MKI project?
Also not sure about expectations here - wondering if 5 seconds are always enough for it to get healthy.

Reply:
Yes, I agree.
I think that this time needs to be way higher and it won't get healthy in 5 seconds.
I'm currently trying to investigate how much time this takes approximately to be within the margin of error.

Reply:
Following @seanrathier review request, I changed the logic to wait for 1 minute and verify the `Pending` or `Healthy` state.